Create ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md (#2101)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-0-0-separated-asm-as-a-plugin-5939.md

---
date: ""

> title: Separated ASM as a plugin. (#5939)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Application security monitoring available as a plugin
```

> title: Separated ASM as a plugin. (#5939)

Reviewer: ```suggestion
title: Separated Advanced software management functionality as a plugin. (#5939)
```

> title: Separated Advanced software management functionality as a plugin. (#5939)

Reviewer 2: ```suggestion
title: Separated Advanced Software Management functionality as a plugin
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3471
version: 1020.0.0
---

> Separated ASM as a plugin. (#5939)

Author: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility in deployments, application security monitoring (ASM) has been separated from the core Dynatrace platform and is now available as a plugin. With this change, ASM can be deployed and updated independently from the Dynatrace platform, allowing for more granular control over the ASM functionality. This separation does not impact existing ASM functionality or configurations, and users can continue to use ASM as before. However, the plugin will need to be installed separately moving forward.
```

> Separated ASM as a plugin. (#5939)

Reviewer: ```suggestion
The Advanced software management logic was previously an integral part of the default Device Management application. With this change, the Advanced software management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#5939)
```

> The Advanced software management logic was previously an integral part of the default Device Management application. With this change, the Advanced software management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. (#5939)

Reviewer 2: ```suggestion
The Advanced Software Management functionality was previously an integral part of the default Device Management application. With this change, the Advanced Software Management service has been restructured to function as a separate plugin. It remains included by default but now offers the flexibility to be added or removed as needed. 
```
